content/en/tracing/guide/tutorial-enable-java-admission-controller.md

```diff
@@ -0,0 +1,276 @@
+kubectl apply -f depl-with-lib-inj.yaml{{< /code-block >}}
+
+6. Run the following command to show that the app and Agent are running:
+   {{< code-block lang="shell" >}}
+kubectl get pods{{< /code-block >}}
+
+    You should see something like this:
+
+    ```
+    NAME                                                READY   STATUS    RESTARTS   AGE
+    datadog-agent-4s8rb                                 3/3     Running   0          28m
+    datadog-agent-cluster-agent-5666cffc44-d8qxk        1/1     Running   0          28m
+    datadog-agent-kube-state-metrics-86f46b8484-mlqp7   1/1     Running   0          28m
+    springback-666db7b6b8-sb4tp                         1/1     Running   0          27m
+    springfront-797b78d6db-mppbg                        1/1     Running   0          27m
+    ```
+
+7. Run the following command to see details of the pod:
+   {{< code-block lang="shell" >}}
+kubectl describe pod springfront{{< /code-block >}}
+
+   You should see something like this:
+
+    ```
+    Events:
+    Type    Reason     Age   From               Message
+    ----    ------     ----  ----               -------
+    Normal  Scheduled  32s   default-scheduler  Successfully assigned default/springfront-797b78d6db-jqjdl to docker-desktop
+    Normal  Pulling    31s   kubelet            Pulling image "gcr.io/datadoghq/dd-lib-java-init:latest"
+    Normal  Pulled     25s   kubelet            Successfully pulled image "gcr.io/datadoghq/dd-lib-java-init:latest" in 5.656167878s
+    Normal  Created    25s   kubelet            Created container datadog-lib-java-init
+    Normal  Started    25s   kubelet            Started container datadog-lib-java-init
+    Normal  Pulling    25s   kubelet            Pulling image "pejese/springfront:v2"
+    Normal  Pulled     2s    kubelet            Successfully pulled image "pejese/springfront:v2" in 22.158699094s
+    Normal  Created    2s    kubelet            Created container springfront
+    Normal  Started    2s    kubelet            Started container springfront
+    ```
+
+    As you can see, an init-container is added to your pod. This container includes the Datadog Java tracing libraries to a volume mount. Also `JAVA_TOOL_OPTIONS` is modified to include `javaagent`. And Datadog-specific environment variables are added to the container:
+
+    ```
+    Environment:
+    DD_ENV:              dev
+    DD_VERSION:          12
+    DD_SERVICE:          springfront
+    DD_ENTITY_ID:         (v1:metadata.uid)
+    DD_TRACE_AGENT_URL:  unix:///var/run/datadog/apm.socket
+    URL:                 http://springback:8088
+    JAVA_TOOL_OPTIONS:    -javaagent:/datadog-lib/dd-java-agent.jar
+    Mounts:
+    /datadog-lib from datadog-auto-instrumentation (rw)
+    /var/run/datadog from datadog (rw)
+    /var/run/secrets/kubernetes.io/serviceaccount from kube-api-access-qvmtk (ro)
+    ```
+
+8. Verify that the Datadog tracing library is injected into the pod by checking the pod logs:
```

Comment:
```suggestion
8. Verify that the Datadog tracing library is injected into the pod by checking the pod logs. For example::
```
I suspect the pod name is randomized, so just calling out that this is an example